libr/core/cmd_open.c

```diff
@@ -1,5 +1,50 @@
 /* radare - LGPL - Copyright 2009-2015 - pancake */
 
+#include <sdb.h>
+
+static void map_list (RIO *io, int mode, RPrint *print) {
+	SdbListIter *iter;
+	RIOMap *map;
+	if (!io || !io->maps || !print || !print->cb_printf)
+		return;
+	ls_foreach_prev (io->maps, iter, map) {
+		switch (mode) {
+			case 1:
```

Comment:
indent case at the same level of switch and put braces around every single line if